modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -345,15 +354,28 @@ def required_claimant_data_is_present
       return if api_version.upcase == 'V1'
 
       # Claimant DOB is always required if they've supplied any claimant headers
-      has_claimant_headers = claimant.birth_date.present?
+      has_claimant_headers = claimant.birth_date.present? && self.class.past?(birth_date) ? true : false
       # form data that includes a claimant is also sufficient to know it's passed the schema
-      has_claimant_data = data_attributes&.fetch('claimant', nil).present?
+      contact_info = claimant.phone_data.present? && claimant.email.present?
+      claimant_exists = claimant.form_data.present? && data_attributes&.fetch('claimant', nil).present?
```

Comment:
What do you think about adding a public `present?` method onto Appellant so that you can check `claimant.present?` here instead of needing to expose the form_data and then do extra fetches on the claimant data from the model too?